src/pages/ProductList/ProductList.js

```diff
@@ -1,8 +1,101 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
+import { Link } from 'react-router-dom';
+import { SUBMENU_LIST, GENDER_LIST, COLOR_LIST, SIZE_LIST } from './filterData';
+
 import './ProductList.scss';
 
 const ProductList = () => {
-  return <div className="productList">ProductList</div>;
+  const [productData, setProductData] = useState([]);
+
+  useEffect(() => {
+    fetch('/data/productListData.json', {
+      method: 'GET',
+    })
+      .then(res => res.json())
+      .then(data => {
+        setProductData(data);
+      });
+  });
+
+  return (
+    <div className="productList">
+      <div className="productListContainer">
+        <div className="filter">
+          <h3 className="menu">남성 신발</h3>
+          <ul className="category">
+            {SUBMENU_LIST.map(({ id, title }) => {
+              return (
+                <Link to="/productList" key={id}>
+                  <li className="categoryMenu">{title}</li>
+                </Link>
+              );
+            })}
```

Comment:
```suggestion

const SUBMENU = SUBMENU_LIST.map(({ id, title }) => {
              return (
                <Link to="/productList" key={id}>
                  <li className="categoryMenu">{title}</li>
                </Link>
              );
            })

return(
<div className="productList">
   <div className="productListContainer">
        <div className="filter">
          <h3 className="menu">남성 신발</h3>
          <ul className="category">
          {SUBMENU}
```
참고참고~~~ 
상태변화없는 상수데이터로 된 단순 UI 구성이라면  변수로도 추출 할 수 있습니다
[공식문서](https://ko.reactjs.org/docs/lists-and-keys.html#embedding-map-in-jsx)를 참고 해보세요~

Reply:
넵 확인했습니다! 
map을 변수로 추출할 수 있는지는 생각 못했는데, 덕분에 앞으로 더욱 가독성 높은 코드를 작성할 수 있겠네요 :)
공식 문서 참고해서 앞으로 코드 작성시 반영해보겠습니다! 👍